TaskManagement/timeline/views.py

```diff
@@ -1,7 +1,23 @@
 from django.shortcuts import render
-from django.views.generic import TemplateView
+from django.contrib.auth.mixins import LoginRequiredMixin
+
+from django.views.generic import View,TemplateView
+from project.models import Project, ProjectTeam
 
 # Create your views here.
 
-class Dashboard(TemplateView):
+class Dashboard(LoginRequiredMixin,TemplateView):
     template_name = "dashboard.html"
+
+    def get(self, request):
+            data = [Project.objects.filter(status=1).count(),Project.objects.filter(status=2).count(),Project.objects.filter(status=3).count()]
+            labels = ["In-Progress","To-Do","Done"]
```

Comment:
you should keep constants in constants.py and i think these are labels used from models itself so you should use DRY concept here.